editoast/src/modelsv2/fixtures.rs

```diff
@@ -0,0 +1,25 @@
+#[cfg(test)]
+pub mod tests {
+    use chrono::Utc;
+
+    use crate::modelsv2::prelude::*;
+    use crate::modelsv2::DbConnection;
+    use crate::modelsv2::Project;
+    use crate::modelsv2::Tags;
+
+    pub fn test_project_changeset(name: &str) -> Changeset<Project> {
```

Comment:
Completely subjective comment + nitpick: I'd drop the `test` prefix as the function module already suggests we're in a test context (we have namespaces for a reason ^^)